woof-code/rootfs-skeleton/usr/sbin/report-video

```diff
@@ -142,7 +155,7 @@ function ScreenColorDepth() {
 
 function get_glxinfo() {
 	if [ `which glxinfo` ]; then 
-		glxinfo 2>/dev/null | grep -E -A2 "direct rendering|OpenGL vendor" | grep -v '\-\-'
+		[ -n "$DISPLAY" ] && glxinfo 2>/dev/null | grep -E -A2 "direct rendering|OpenGL vendor" | grep -v '\-\-'
```

Comment:
Bot DISPLAY and WAYLAND_DISPLAY are set under Wayland

Reply:
Not pure wayland, where `glxinfo` fails.